cmd/upload/upload.go

```diff
@@ -161,6 +162,10 @@ func newCommand(ctx context.Context, common *cmd.SharedFlags, args []string, fsO
 		"discard-archived",
 		" google-photos only: Do not import archived photos (default FALSE)", myflag.BoolFlagFn(&app.DiscardArchived, false))
 
+	cmd.BoolFunc(
+		"auto-archive",
+		" google-photos only: Automatically archive photos that are also archived in google photos (default FALSE)", myflag.BoolFlagFn(&app.AutoArchive, true))
```

Comment:
The description claims the default is false while it looks like the code defaults to true.

Reply:
Good catch, I missed that when I updated the default to true